Pipfile

```diff
@@ -7,6 +7,7 @@ aiohttp = ">=2.2.3,<2.3.0"
 async-timeout = ">=1.2.1,<1.3.0"
 chardet = ">=3.0.4,<3.1.0"
 multidict = ">=3.1.3,<3.2.0"
+pyyaml = ">=3.12,<3.13"
```

Comment:
PyYAML is outdated - [ruamel.yaml](https://pypi.python.org/pypi/ruamel.yaml) is better